docroot/themes/contrib/civictheme/civictheme_library/components/00-base/_variables.components.scss

```diff
@@ -883,31 +883,31 @@ $ct-accordion-dark-wrapper-background-color: ct-color-dark('highlight') !default
 //
 // Alert.
 //
-$ct-alert-light-divider-color: ct-color-light('background') !default;
-$ct-alert-light-error-background-color: ct-color-light('error') !default;
-$ct-alert-light-error-color: ct-color-light('background') !default;
-$ct-alert-light-error-icon-color: $ct-alert-light-error-color !default;
-$ct-alert-light-information-background-color: ct-color-light('information') !default;
-$ct-alert-light-information-color: ct-color-light('background') !default;
-$ct-alert-light-information-icon-color: $ct-alert-light-information-color !default;
-$ct-alert-light-success-background-color: ct-color-light('success') !default;
-$ct-alert-light-success-color: ct-color-light('background') !default;
-$ct-alert-light-success-icon-color: $ct-alert-light-success-color !default;
-$ct-alert-light-warning-background-color: ct-color-light('warning') !default;
-$ct-alert-light-warning-color: ct-color-light('background') !default;
-$ct-alert-light-warning-icon-color: $ct-alert-light-warning-color !default;
+$ct-alert-light-divider-color: ct-color-light('interaction-background') !default;
+$ct-alert-light-error-background-color: ct-color-tint(ct-color-light('error'), 90) !default;
+$ct-alert-light-error-color: ct-color-light('body') !default;
+$ct-alert-light-error-icon-color: ct-color-light('error') !default;
+$ct-alert-light-information-background-color: ct-color-tint(ct-color-light('information'), 90) !default;
+$ct-alert-light-information-color: ct-color-light('body') !default;
+$ct-alert-light-information-icon-color: ct-color-light('information') !default;
+$ct-alert-light-success-background-color: ct-color-tint(ct-color-light('success'), 90) !default;
+$ct-alert-light-success-color: ct-color-light('body') !default;
+$ct-alert-light-success-icon-color: ct-color-light('success') !default;
+$ct-alert-light-warning-background-color: ct-color-tint(ct-color-light('warning'), 90) !default;
+$ct-alert-light-warning-color: ct-color-light('body') !default;
+$ct-alert-light-warning-icon-color: ct-color-light('warning') !default;
 $ct-alert-dark-divider-color: ct-color-light('interaction-background') !default;
-$ct-alert-dark-error-background-color: ct-color-light('error') !default;
-$ct-alert-dark-error-color: ct-color-light('background') !default;
+$ct-alert-dark-error-background-color: ct-color-shade(ct-color-dark('error'), 30) !default;
```

Comment:
Updated to use opacity from message. 
Note figma for alert for dark has 80% but as per message its 70%.
Updated colors as per message component.